app/src/main/java/com/beemdevelopment/aegis/ui/views/EntryListView.java

```diff
@@ -84,22 +83,33 @@ public long getMillisTillNextRefresh() {
     }
 
     public void setGroupFilter(String group) {
```

Comment:
Ah, yeah that's still there because I based it on the two setSortCategory functions. I'll remove both of the ones that don't have an 'apply' parameter.